src/components/basic/Card.md

```diff
@@ -47,15 +49,17 @@ import Section from "./containers/Section";
             role="img"
             aria-label="Placeholder: Thumbnail"
           >
-            <rect width="100%" height="200px" fill="#C2C2C2"></rect>
+            <rect width="100%" height="150px" fill="#C2C2C2"></rect>
```

Comment:
it looks like this placeholder svg is being repeated over and over. This sounds like a good instance to create a small component within this file and then reference that.

Example:

```jsx
const ImagePlaceholder = props => <svg><rect ></rect></svg>;
```

Then you can use it in the cards, like so

```jsx
<CardContent>
  <ImagePlaceHolder />
...
```



Then in the future, if we have to tweak the svg, we only need to do it in one place.

Reply:
done